docs/user/intro/docusaurus.rst

```diff
@@ -0,0 +1,97 @@
+
+Docusarus
+=========
+
+.. meta::
+   :description lang=en: Hosting Docusaurus sites on Read the Docs.
+
+`Docusaurus`_ is a static-site generator that builds a single-page application with fast client-side navigation and out-of-the-box documentation features.
+
+Minimal configuration required to build a Docusaurus project on Read the Docs looks like this,
+specifying a nodejs toolchain on Ubuntu, using multiple :ref:`build <config-file/v2:build>` commands to install the requirements,
+build the site, and copy the output to $READTHEDOCS_OUTPUT:
+
+.. code-block:: yaml
+   :caption: .readthedocs.yaml
+
+    version: 2
+    build:
+    os: "ubuntu-22.04"
+    tools:
+        nodejs: "18"
+    commands:
+        # "docs/" was created following the Docusaurus tutorial:
+        # npx create-docusaurus@latest docs classic
+        # but you can just use your existing Docusaurus site
+        #
+        # Install Docusaurus dependencies
+        - cd docs/ && npm install
+        # Build the site
+        - cd docs/ && npm run build
+        # Copy generated files into Read the Docs directory
+        - mkdir --parents $READTHEDOCS_OUTPUT/html/
+        - cp --recursive docs/build/* $READTHEDOCS_OUTPUT/html/
+
+.. _Docusaurus: https://docusaurus.io/
+
+Supported Features
+------------------
+
+.. csv-table:: Supported Features
+   :header: "Feature", "Description", "Supported"
+
+   "Pull request previews", "Preview changes to your documentation before merging.", "✅"
+   "Versioning", "Supports multiple versions of your documentation.", "✅"
+   "Flyout menu", "Provides a flyout menu for navigation.", "✅"
+   "Search", "Provides full-text search capabilities.", "❌"
+   "Offline formats", "Generates PDF and ePub formats.", "❌"
+   "Localization", "Supports multiple languages.", "❌"
```

Comment:
What's the limitation that doesn't allow us to support translations?

Reply:
There isn't really support for translation in terms of the build system? Though I guess we removed that from Sphinx builds too now, haven't we?

Reply:
> terms of the build system

What does this mean?

> Though I guess we removed that from Sphinx builds too now, haven't we?

We are not doing anything special in Sphinx for translations, no.

---

So, basically if they import a project and set it as a translation, we supported it. No matter what tool they are using.